app/controllers/admin/organisations_controller.rb

```diff
@@ -14,9 +15,18 @@ def new
     @organisation = Organisation.new
   end
 
+  def show
+    @adapter = @organisation.try(:adapter)
+  end
+
   def edit
     @organisation = Organisation.find(params[:id])
     @adapter = @organisation.try(:adapter)
+    if @adapter.present?
+      country_ids = @adapter.countries_with_access_ids
+      @selected_countries_with_access = Country.where(id: country_ids).order(:name).
```

Comment:
we could avoid querying for countries here if @adapter had a method /association to return countries_with_access